app/config/AppConfig.scala

```diff
@@ -37,14 +37,21 @@ class AppConfig @Inject()(configuration: Configuration, servicesConfig: Services
   val variationsApiSchema4MLD: String = "/resources/schemas/4MLD/variations-api-schema-4.0.json"
   val variationsApiSchema5MLD: String = "/resources/schemas/5MLD/variations-api-schema-4.2.0.json"
 
+  private def insertTRN(url: String, trn: String) = url.replace(":trn", trn)
+
   val taxEnrolmentsPayloadBodyServiceName : String =
     configuration.get[String]("microservice.services.tax-enrolments.serviceName")
```

Comment:
Could you make this path similar to non-taxable and include taxable in the path?

Reply:
I'd also suggest adding taxable into the value name

Reply:
Done